js/blocks/controls/email.js

```diff
@@ -4,6 +4,8 @@ const { TextControl } = wp.components;
 
 const BlockLabEmailControl = ( props ) => {
 	const { field, getValue, onChange } = props;
+	const initialValue = getValue( props );
+	const value = 'undefined' !== typeof initialValue ? initialValue : field.default;
```

Comment:
Great solution. Really glad to get this console error sorted.

Reply:
Thanks!

Reply:
Thanks!